client/benchmark/build.gradle

```diff
@@ -17,8 +17,7 @@ tasks.named("assemble").configure {  enabled = true }
 archivesBaseName = 'client-benchmarks'
 mainClassName = 'org.elasticsearch.client.benchmark.BenchmarkMain'
 
-// never try to invoke tests on the benchmark project - there aren't any
-tasks.named("test").configure {enabled = false }
```

Comment:
I removed those explicit set tests to not execute. Gradle will shortcut that anyhow as no source is detected. It also makes the outcome of that task more clear IMO as it will tell `NO_SOURCE` instead of SKIPPED which can have different reasons.